routes/main_routes.js

```diff
@@ -85,6 +85,7 @@ module.exports = function (app, passport) {
                         errorMessage: req.flash('error')
                     });
                 } else {
+                    res.header('token', passportLib.generateJwtToken(req.body.email));
```

Comment:
let's add a middleware handler for it
```.use(function timeLog (req, res, next) {
  console.log('Time: ', Date.now())
  next()
})```

- checks for cookie, if exists and validated next()
- otherwise redirect to login, login should have a callback param to know where to return to